File: src/main/java/com/telegram/folobot/repos/FoloVarRepo.java
package com.telegram.folobot.repos;

import com.telegram.folobot.domain.FoloVar;
import com.telegram.folobot.domain.FoloVarId;
import org.springframework.data.repository.CrudRepository;

import java.util.List;

public interface FoloVarRepo extends CrudRepository<FoloVar, FoloVarId> {
    List<FoloVar> findByChatidAndType(long chatid, String type);
}

File: src/main/kotlin/com/everbald/folobot/persistence/repos/FoloVarRepo.kt
package com.everbald.folobot.persistence.repos

import com.everbald.folobot.persistence.entity.FoloVarEntity
import com.everbald.folobot.persistence.entity.FoloVarId
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface FoloVarRepo : CrudRepository<FoloVarEntity, FoloVarId> {
    fun findVarById(foloVarId: FoloVarId): FoloVarEntity?
}